src/components/HomePage.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import MovieCard from "./MovieCard";
+import img from "../Asset/undraw.png";
+import { Container } from "../styles/Container.styled";
+import GlobalStyles from "../styles/GlobalStyles";
+import { Cards } from "../styles/Cards.styled"
+import { Empty } from "../styles/Empty.styles";
+
+const key = 'api_key=06431b5693dfdfb9f2a94311f7d8cbf0';
```

Comment:
Use environment variables to store these secrets and base url - https://create-react-app.dev/docs/adding-custom-environment-variables/